src/components/CartModal/CartModal.tsx

```
@@ -0,0 +1,49 @@
+          {cartItemData?.map(({ imageUrl, name, price, cartInfo }) => (
+            <ItemModalCard
+              imageUrl={imageUrl}
+              name={name}
+              price={price * cartInfo.quantity}
+              quantity={cartInfo.quantity}
+              onRemoveCart={() => handleCartProducts('remove', { id: cartInfo.id })}
+              onPatchCart={(quantity: number) =>
+                handleCartProducts('patch', { id: cartInfo.id, quantity })
+              }
+            />
+          ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)